.prettierignore

```diff
@@ -2,3 +2,5 @@ generated
 coverage
 build
 dist
+package-lock.json
+node_modules
```

Comment:
Please also add a directory with Storybook build output. Tested that locally and it works. Thanks!

```suggestion
node_modules
storybook-static
```